### PR TITLE
Contrary to our configuration, Chinese doesn't use plurals

### PR DESCRIPTION
**Pull request by mathjazz**
For more details, see:
http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html#zh
http://localization-guide.readthedocs.org/en/latest/l10n/pluralforms.html#f2

This PR resets Chinese plurals configuration and removes Chinese translations in plural forms.

@Osmose, r?

### CODE REVIEW

File: pontoon/base/migrations/0018_auto_20150820_0925.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+def change_chinese_plurals(apps, schema_editor):
+    Locale = apps.get_model('base', 'Locale')
+    Translation = apps.get_model('base', 'Translation')
+
+    for locale in Locale.objects.filter(name="Chinese"):
+        locale.nplurals = 1
+        locale.plural_rule = '0'
+        locale.cldr_plurals = '5'
```
Comment (Osmose) [inline pontoon/base/migrations/0018_auto_20150820_0925.py]: I don't know much about CLDR, so I'm assuming this is fine, but I'm curious: why this is set to 5?

File: pontoon/base/migrations/0018_auto_20150820_0925.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        locale.save()
+
+        # Delete plural translations
+        Translation.objects.filter(locale=locale, plural_form=1).delete()
```
Comment (Osmose) [inline pontoon/base/migrations/0018_auto_20150820_0925.py]: `locale.translation_set` should work here too.

**Comment by Osmose** [comment]
r+, looks good!

File: pontoon/base/migrations/0018_auto_20150820_0925.py (hunk 1 above)
Comment (mathjazz) [inline pontoon/base/migrations/0018_auto_20150820_0925.py]: 5 is for other:
http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html#zh

It's weird, I know.

**Pull request merged**